slack_sdk/web/async_client.py

```diff
@@ -1601,6 +1601,83 @@ async def auth_test(
         """
         return await self.api_call("auth.test", params=kwargs)
 
+    async def bookmarks_add(
+        self,
+        *,
+        channel_id: str,
+        title: str,
+        type: str,
+        emoji: Optional[str] = None,
+        entity_id: Optional[str] = None,
+        link: Optional[str] = None,
```

Comment:
We could add a comment here: 
```suggestion
        link: Optional[str] = None,  # include when type is 'link'
```